spring-integration-core/src/main/java/org/springframework/integration/handler/advice/RequestHandlerCircuitBreakerAdvice.java

```diff
@@ -99,12 +102,12 @@ private AtomicInteger getFailures() {
 	/**
 	 * An exception thrown when the circuit breaker is in an open state.
 	 */
-	public static final class CircuitBreakerOpenException extends RuntimeException {
+	public static final class CircuitBreakerOpenException extends MessagingException {
 
 		private static final long serialVersionUID = 1L;
 
-		CircuitBreakerOpenException(String message) {
-			super(message);
+		public CircuitBreakerOpenException(Message<?> message, String description) {
```

Comment:
Doh. That is artifact of the automatic IDEA refactoring. Won't mind if you fix on merge.

Thanks